src/pages/nft/class/_classId/_nftId.vue

```diff
@@ -481,6 +562,9 @@ export default {
         1
       );
     },
+    handleClickViewMessage() {
+      scrollToTop();
+    },
```

Comment:
I think this can achieve by adding `scrollToTop: true` to message page

https://nuxtjs.org/docs/components-glossary/scrolltotop#the-scrolltotop-property